src/app/components/applications/SSV/MyAccount/components/SingleOperator/SingleOperator.tsx

```diff
@@ -336,14 +330,14 @@ const SingleOperator = () => {
             </Grid>
           </Grid>
         )}
-        {validators_count > 0 && (
+        {validators_count! > 0 && (
           <Grid item className={classes.OperatorsValidatorsTable}>
             <Table
               data={data}
               columns={columns}
               actionProps={{
                 onChangePage,
-                perPage: per_page,
+                perPage: per_page as number,
```

Comment:
@Chris-ssvlabs just make sure this will not break anything if the perPage is undefined because it can be, see line 84

Reply:
There are not changes function wise